server/api/catalogs/routes.py

```diff
@@ -54,3 +60,46 @@ async def get_catalog(siret: Siret) -> CatalogView:
         return await bus.execute(GetCatalogBySiret(siret=siret))
     except CatalogDoesNotExist as exc:
         raise HTTPException(404, detail=str(exc))
+
+
+export_cache: Dict[str, Tuple[dt.datetime, Any]] = {}
```

Comment:
(Petit détail) On devrait peut-être nommer cette variable comme ceci :

```suggestion
_EXPORT_CACHE: Dict[str, Tuple[dt.datetime, Any]] = {}
```

En effet c'est la convention habituelle pour 1) une variable définie au top-level (pas dans une fonction), 2) une variable pas destinée à être accédée de l'extérieur par un import.

En effet dans `export_catalog()`, la lgine `if siret in export_cache` est un peu surprenante, on se demander "d'où vient cette variable qui n'est pas définie dans cette fonction ?". `if siret in _EXPORT_CACHE` indique mieux où regarder.